lib/services.js

```diff
@@ -478,6 +479,7 @@ export default {
         const mpuBucketName = `${constants.mpuBucketPrefix}${bucketName}`;
         metadata.getBucket(mpuBucketName, log, (err, mpuBucket) => {
             if (err) {
+                log.error('error from metadata', { error: err });
```

Comment:
Here is the original log: https://github.com/scality/S3/blob/master/lib/metadata/wrapper.js#L50
I think that in this case, we're right to log an error now, as this bucket SHOULD exist and be accessible, due to the fact that we're already validating a multipart upload. But then, maybe we should log any error != 404 in the lower layer (wrapper or even lower)